kentik_api_library/pyproject.toml

```diff
@@ -0,0 +1,110 @@
+[build-system]
+requires = ["setuptools", "wheel", "setuptools_scm", "gitpython"]
+build-backend = "setuptools.build_meta"
+
+[tool.black]
+line-length = 120
+target-version = ['py39']
+
+[tool.isort]
+line_length = 120
+profile = "black"
+known_local_folder = ["kentik_api", "examples", "tests"]
+
+[tool.mypy]
+ignore_missing_imports = true
+exclude = "(generated|build)/"
+
+[[tool.mypy.overrides]]
```

Comment:
Are the double square brackets required?

Reply:
I think the whole section is redundant. Removing